generate.py

```diff
@@ -8,6 +8,9 @@
 from utils.helpers import get_new_literal_structures, reset_new_literal_structures, StructureKind
 
 
+BITWISE_ENUMS = ['WatchKind']
```

Comment:
maybe move this to generate_enumerations file?

Reply:
nah, it is ok here.

Reply:
I wasn't sure myself how generic the generation code is meant to be (whether it should have knowledge of specific LSP types)